index.html

```diff
@@ -2663,8 +2807,192 @@ <h3>#xmlLang-root</h3>
+                has opened such a pull request,
+                which MUST represent the requested changes
+                and MUST be linked to a related issue.</li>
+              <li>The TTWG follows its Decision Policy to review
+                the proposal in the pull request.</li>
+              <li>At the end of the Decision Review Period
+                if a TTWG Chair declares that there is consensus to approve,
+                the <a>change request</a> is approved.</li>
+              <li>In the absence of consensus to approve the expectation is
+                that a discussion will happen, to include the
+                change requester.
+                The result of this discussion can be any one of:
+                <ol>
+                  <li>the <a>change request</a> is abandoned;</li>
+                  <li>the <a>change request</a> is modified for another review;</li>
+                  <li>if the discussion resolves the objections,
+                    and a TTWG Chair declares consensus to approve,
+                    the <a>change request</a> can be approved.</li>
+                </ol>
+              </li>
+            </ul>
+            <p>An approved <a>change request</a> is enacted by merging its
+              related pull request into the <a class="pp-vcs" href="">version control system</a>
+              and publishing the updated version of this document.</p>
+          </section>
+          <section id="team-change-request-assessment">
+            <h5>Custodian is the W3C Team</h5>
+            <p>If the <a>custodian</a> is the <a>W3C Team</a>,
+              the Team MUST seek <a>wide review</a> of the <a>change request</a> and
+              offer a review period of at least 4 weeks,
+              before assessing from the responses received
+              if there is consensus amongst the respondents.</p>
+            <p>The Team MAY require a pull request on the <a class="pp-vcs" href="">version control system</a>
+              to be opened as the basis of the review.
+            </p>
+            <p>If there is such consensus,
+              the Team MUST make the proposed changes.</p>
+          </section>
+        </section>
+      </section>
+      <h3>Registry Table Constraints</h3>
+      <p>This section defines constraints on the <a>registry tables</a>
+        defined in this document.
+        Each <a>registry table</a> consists of a set of
+        <dfn data-cite="w3c-process#registry-entry" data-lt="registry entry|registry entries">registry entries</dfn>.</p>
+      <section id="registry-entries">
+        <h4>Registry Entries</h4>
+        <p>Each <a>registry entry</a> has a <a>status</a>, a unique <dfn>key</dfn>, and
+          if appropriate, other <dfn>fields</dfn>, for example any notes,
+          a description, or a reference to some other
+          defining entity.
+        </p>
+        <p>The <a>registry table</a> definition MUST define
```

Comment:
For here, isn't it better to link as `registry table definition` but not `registry table`? since MUST targets definition but not table itself...

Reply:
I think you're right, that would be better. Will look to see if it causes any reference problems, and fix if I can.

Reply:
Done in 03ab6c7.